packages/tools/devtools/devtools-browser-extension/src/content/ContentScript.ts

```diff
@@ -18,6 +18,10 @@ import {
 
 type Port = chrome.runtime.Port;
 
+// `window` should always be defined in the Content script context.
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
I lean towards an explicit check and throwing an understandable error if `maybeWindow` happens to be undefined in here, vs something later throwing "cannot find property X of undefined". I agree it should never happen (barring browsers doing something very weird to their extensions model), but I'd argue code size for the extension is not as critical as for the framework itself, so an explicit error is preferable.